docs/Pitfalls.asciidoc

```diff
@@ -53,3 +53,45 @@ impacted by snapshot creation. Loading VM snapshots can also have an impact on
 SUT behavior as the execution of the first step after loading a snapshot might
 be delayed. This can lead to problems if the executed tests do not foresee an
 appropriate timeout margin.
+
+== DB migration from sqlite to postgreSQL
+In order to start using postgreSQL DBMS with openQA one need to perform following steps:
+- Install postgresql-server.
+- Adjust postgresql configuration file,
+namely `/var/lib/pgsql/data/pg_hba.conf`. For example:
+
+```
+local   all             all                                     md5
+local   all             postgres                                md5
+local   all             openqa                                  md5
+# IPv4 local connections:
+host    all             all             127.0.0.1/32            md5
+# IPv6 local connections:
+host    all             all             ::1/128                 md5
+
+```
+- Connect to the DBMS (`sudo sudo -u postgres psql`) and create user, db and delegate
+permissions to that user:
+
+```
+CREATE DATABASE openqa;
+CREATE USER openqa;
+GRANT ALL PRIVILEGES ON DATABASE openqa to openqa;
+ALTER USER openqa PASSWORD 'nots3cr3t';
+```
```

Comment:
What would that be necessary for? That seems to duplicate in a different way of writing what is already mentioned on https://github.com/os-autoinst/openQA/blob/master/docs/Contributing.asciidoc#how-to-setup-postgresql-to-test-locally-with-production-data . I suggest to not duplicate that but rather extract the common part into the installation guide and reference from there more specific issue descriptions and workarounds.

Reply:
Yeah, I remembered that I saw this somewhere, but expected it to be in installation guide.

Reply:
I would replace these instructions for:

 ```systemctl enable postgresql.service
 systemctl start postgresql.service
 su - postgres -c 'createuser -D geekotest'
 su - postgres -c 'createdb -O geekotest openqa'```

Reply:
most of this is really not necessary for a local user - you want to use peer auth over unix sockets, not passwords over tcp.